samples/model-builder/test_constants.py

```diff
@@ -149,3 +149,19 @@
 MIN_REPLICA_COUNT = 1
 MAX_REPLICA_COUNT = 1
 ENDPOINT_DEPLOY_METADATA = ()
+MODEL_SERVING_CONTAINER_COMMAND = (["/usr/bin/tensorflow_model_server"],)
+MODEL_SERVING_CONTAINER_ARGS = (
+    [
+        f"--model_name={MODEL_NAME}",
+        "--model_base_path=$(AIP_STORAGE_URI)",
+        "--rest_api_port=8080",
+        "--port=8500",
+        "--file_system_poll_wait_seconds=31540000",
+    ],
+)
+MODEL_SERVING_CONTAINER_PREDICT_ROUTE = (f"/v1/models/{MODEL_NAME}:predict",)
+MODEL_SERVING_CONTAINER_HEALTH_ROUTE = f"/v1/models/{MODEL_NAME}"
```

Comment:
Do we use these? They look like the partial path used to call the container via REST.

Reply:
Removed